numpy/core/tests/test_multiarray.py

```diff
@@ -6517,6 +6517,7 @@ def test_max_dims(self):
         a = np.empty((1,) * 32)
         self._check_roundtrip(a)
 
+    @pytest.mark.skipif(sys.version_info < (2, 7, 7), reason="See gh-11115")
```

Comment:
I'd prefer just `relies on a bpo-10744 being fixed`

Reply:
@eric-wieser The Python issue seems a bit less discoverable and the numpy reference has a fair amount of useful discussion. Given that we will drop 2.7 support at the end of the year, probably not worth spending much time on this. If nothing else, git blame will lead back here.